client/src/services/authCalls.tsx

```diff
@@ -0,0 +1,78 @@
+import axios from "axios";
+import { userType } from "./types";
+axios.defaults.withCredentials = true;
```

Comment:
What is the function of this code snippet?

Reply:
the backend uses cookies to store the credentials for frontend so we use this

Reply:
Ok will explore on it to understand its utility. Thanks @cjinc121 !!